frontend/src/components/Layout/Navbar.tsx

```diff
@@ -16,7 +19,12 @@ const Navbar = () => {
   const darkMode = useDarkModeStore((state) => state.darkMode);
   const { t } = useTranslation(['routes']);
   const auth = useSafeAuth();
-
+  const backendVersion = sessionStorage.getItem('version');
+  useEffect(() => {
+    if (backendVersion != 'frontendVersion') {
+      toast(`Ihre Version ist nicht mehr aktuell. Bitte laden Sie die Seite neu.`);
```

Comment:
You can translate the string using useTranslation

Reply:
Oh thats a typo by me, shouldnt be a string